sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -408,13 +405,14 @@ export class MessageSender extends LinkEntity {
         }
       });
 
-    const jitterInSeconds = randomNumberFromInterval(1, 4);
```

Comment:
@chradek We had some talks in Event Hubs of re-introducing jitter. That should probably apply here as well right?

Reply:
In event hubs we were talking about adding jitter for load balancing to try and reduce the chance for contention when claiming partitions. I don't remember why it would have gotten removed here though...

Reply:
Logged an issue https://github.com/Azure/azure-sdk-for-js/issues/7351 upon discussing with @chradek